Tests/Factory/HttpFoundationFactoryTest.php

```diff
@@ -47,7 +48,7 @@ public function testCreateRequest()
             new Stream('The body'),
             '/about/kevin',
             'GET',
-            'http://les-tilleuls.coop/about/kevin',
+            new Uri('http://les-tilleuls.coop/about/kevin'),
```

Comment:
Can you add another test instead of editing this one. So we can assert that `string` and `Uri` are supported.